src/node/hooks/express/importexport.js

```diff
@@ -3,8 +3,22 @@ var settings = require('../../utils/Settings');
 var exportHandler = require('../../handler/ExportHandler');
 var importHandler = require('../../handler/ImportHandler');
 var padManager = require("../../db/PadManager");
+var authorManager = require("../../db/AuthorManager");
+const rateLimit = require("express-rate-limit");
+
+settings.importExportRateLimiting.onLimitReached = function(req, res, options) {
+  // when the rate limiter triggers, write a warning in the logs
+  const ip = req.headers["x-forwarded-for"] || req.connection.remoteAddress;
```

Comment:
This is not necessary (to be verified...): `req.ip` should already have the correct value once `settings.trustProxy` is set.

See: https://expressjs.com/en/guide/behind-proxies.html

Reply:
Ok verified.